src/Redmine/Client/ClientApiTrait.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Redmine\Client;
+
+use InvalidArgumentException;
+use Redmine\Api;
+
+/**
+ * client api trait.
```

Comment:
not very helpful, i would remove it

Reply:
I rephrased it and add an `@internal` docblock. What do you think?